core/controlplane/config/templates/cloud-config-worker

```diff
@@ -198,10 +202,14 @@ coreos:
         Type=oneshot
         RemainAfterExit=true
         ExecStart=/bin/true
-        TimeoutStopSec=30s
+        TimeoutStopSec=60s
         ExecStop=/bin/sh -c '/usr/bin/rkt run \
         --volume=kube,kind=host,source=/etc/kubernetes,readOnly=true \
         --mount=volume=kube,target=/etc/kubernetes \
+        {{- if .Experimental.ClusterTLSBootstrap.Enabled }}
+        --volume=kuberun,kind=host,source=/var/run/kubernetes,readOnly=true \
+        --mount=volume=kuberun,target=/var/run/kubernetes \
+        {{- end }}
```

Comment:
If we go [that](https://github.com/kubernetes-incubator/kube-aws/pull/449/files#r108824744) way, this block could be just:

```
--volume=kuberun,kind=host,source=/var/run/kubernetes,readOnly=true \
--mount=volume=kuberun,target=/var/run/kubernetes \
```

without a `if`.